### ISSUE TITLE
GraphConstructor and BlieprintGraphOutputFormat

### ISSUE THREAD

**Issue by okram** [thinkaurelius/faunus#125]
BlueprintsGraphOuptputFormat has a hardcoded "graph constructor". To support arbitrary graph referencing and graph add-ons like wrappers, see about using a Gremlin script to generate the graph object or a META-INF/servo es model.

**Comment by okram** [comment]
GraphFactory could be used here perhaps? @spmallette

**Comment by spmallette** [comment]
yes...we could replace with GraphFactory when we bump to 2.4.0.  i'll take this one.

**Comment by okram** [comment]
@spmallette  Faunus has been bumped to 2.4.0.
